src/base-importer/BaseImporter.js

```diff
@@ -0,0 +1,358 @@
+import RequestContext, { FileReference, Auth } from 'api-flow'
+
+import {
+    registerImporter,
+    DynamicValue,
+    DynamicString
+} from '../paw-mocks/PawShims'
+
+@registerImporter // eslint-disable-line
```

Comment:
I would remove `@registerImporter` from here. Otherwise, subclasses will register again and we will end up having twice the call, which would break Paw's API.